packages/live-status-gateway/api/schemas/packages.yaml

```diff
@@ -0,0 +1,63 @@
+title: Packages
+description: Packages schema for websocket subscriptions
+$defs:
+  packages:
+    type: object
+    properties:
+      event:
+        type: string
+        const: packages
+      rundownPlaylistId:
+        description: Unique id of the rundown playlist, or null if no playlist is active
+        oneOf:
+          - type: string
+          - type: 'null'
+      packages:
+        description: The Package statuses for this playlist
+        type: array
+        items:
+          $ref: '#/$defs/package'
+    required: [event, rundownPlaylistId, packages]
+    additionalProperties: false
+    examples:
+      - event: packages
+        rundownPlaylistId: 'OKAgZmZ0Buc99lE_2uPPSKVbMrQ_'
+        packages:
+          $ref: '#/$defs/package/examples'
+  package:
+    type: object
+    properties:
+      packageName:
+        description: Name of the package
+        oneOf:
+          - type: string
+          - type: 'null'
+      statusCode:
+        description: Status code
+        type: number
+      rundownId:
+        description: Id of the Rundown that expects this package
+        type: string
+      partId:
+        description: Id of the Part that expects this package
+        type: string
+      segmentId:
+        description: Id of the Segment that expects this package
+        type: string
+      pieceId:
```

Comment:
I think this should have a more descriptive name? This isn't neccessarily the pieceId, this is effectively either an AdLib or a Piece (or Piece Instance!), so this name should reflect that, IMO.